packages/pegasus/src/pegasus.js

```diff
@@ -457,9 +457,15 @@ const makePegasus = (zcf, board, namesByAddress) => {
      * @param {ERef<Peg>} pegP the peg over which to transfer
      * @param {DepositAddress} depositAddress the remote receiver's address
      * @param {string} [memo] the memo to attach to ics transfer packet
+     * @param {SenderOptions} [opts] additional sender options
      * @returns {Promise<Invitation>} to transfer, make an offer of { give: { Transfer: pegAmount } } to this invitation
      */
-    async makeInvitationToTransfer(pegP, depositAddress, memo = '') {
+    async makeInvitationToTransfer(
+      pegP,
+      depositAddress,
+      memo = '',
+      opts = { sender: DEFAULT_SENDER_ADDRESS },
+    ) {
```

Comment:
Let's not expose DEFAULT_SENDER_ADDRESS at this layer.
```suggestion
      opts = /** @type {SenderOptions} */ ({}),
    ) {
```